3-enrich/stream-enrich/core/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/model.scala

```diff
@@ -91,12 +91,14 @@ object model {
     aws: AWSCredentials,
     gcp: Option[GCPCredentials],
     region: String,
-    maxRecords: Int,
+    maxRecords: Option[Int],
```

Comment:
making this optional because enhanced fan out doesn't use it. if we supply it, it will enable regular mode.